front/src/applications/stdcm/components/StdcmForm/StdcmOrigin.tsx

```diff
@@ -33,8 +33,14 @@ const StdcmOrigin = ({
     () => ({
       originArrival: origin?.arrival ? extractDateAndTimefromISO(origin.arrival) : undefined,
       originToleranceValues: {
-        arrivalToleranceBefore: origin?.arrivalToleranceBefore || DEFAULT_TOLERANCE,
-        arrivalToleranceAfter: origin?.arrivalToleranceAfter || DEFAULT_TOLERANCE,
+        arrivalToleranceBefore:
+          origin?.arrivalToleranceBefore === undefined
+            ? DEFAULT_TOLERANCE
+            : origin.arrivalToleranceBefore,
+        arrivalToleranceAfter:
+          origin?.arrivalToleranceAfter === undefined
+            ? DEFAULT_TOLERANCE
+            : origin.arrivalToleranceAfter,
```

Comment:
Why these changes ? I think the previous version does the same thing and is more concise